docker/xsoar-tools/Pipfile

```diff
@@ -5,7 +5,7 @@ name = "pypi"
 
 [packages]
 demisto-py = "*"
-demisto-sdk = "==1.17.2" # this package should be updated manually. No reason this is locked
+demisto-sdk = "*"
```

Comment:
didnt we want this locked ? @dorschw

Reply:
indeed, it should be locked.

Reply:
why is it locked? why can't we update it with depandabot?

Reply:
we can, but want to control it manually.

Reply:
hence the comment

Reply:
@adi88d i still dont understand why we want to control it manually but go ahead and lock it.
i don't see any reason why we need to update it manually each time, it makes some of our content scripts to run with old SDK versions which contain bugs that were fixed in later versions

Reply:
See [here](https://panw-global.slack.com/archives/C03L972AER1/p1692798368468139)